test/OpenTelemetry.Tests/Trace/ActivityExtensionsTest.cs

```diff
@@ -74,20 +74,6 @@ public void SetCancelledStatus()
             Assert.True(activity.GetStatus().StatusCode.Equals(Status.Error.StatusCode));
         }
 
-        [Fact]
-        public void GetStatusWithNoStatusInActivity()
```

Comment:
we need to make `Unset` as default, and a test to validate that the status is `Unset`

Reply:
just enabled again and tested. Added a new test comparing `default` with `Status.Unset`